embassy-net/src/raw.rs

```diff
@@ -5,10 +5,10 @@ use core::mem;
 use core::task::{Context, Poll};
 
 use embassy_net_driver::Driver;
-use smoltcp::iface::{Interface, SocketHandle};
+pub use smoltcp::iface::{Interface, SocketHandle};
```

Comment:
why do you need these? they're not used anywherein the RawSocket public API.

Reply:
I mistook `RawSocket::with_mut` to be public. So these aren't required, I'll remove them when I get home.